tests/system/Log/ChromeLoggerHandlerTest.php

```diff
@@ -0,0 +1,28 @@
+<?php namespace CodeIgniter\Log\Handlers;
+
+use Config\MockLogger as LoggerConfig;
+use Psr\Log\LogLevel;
+use CodeIgniter\Log\Handlers\TestHandler;
+
+class ChromeLoggerHandlerTest extends \CIUnitTestCase
+{
+	public function setUp()
+	{
+	}
+
+	//--------------------------------------------------------------------
+
+        public function testCanHandleLogLevel()
+	{
+		$config = new LoggerConfig();
+                $config->handlers['CodeIgniter\Log\Handlers\TestHandler']['handles'] =  ['critical'];
+		//$config->threshold = "foo";
+                
+		$logger = new ChromeLoggerHandler($config->handlers['CodeIgniter\Log\Handlers\TestHandler']);
```

Comment:
Is this testing something specific to ChromeLoggerHandler or is this a feature of the BaseConfig? If so, it should be tested there.